packages/orbit-components/src/primitives/TooltipPrimitive/components/TooltipContent.js.flow

```diff
@@ -15,9 +16,9 @@ export type Props = {|
   onClose: () => void,
   onCloseMobile: () => void,
   onEnter: () => void,
-  preferredPosition: ?Positions,
-  preferredAlign: ?Aligns,
-  containerRef: React.ElementRef<*>,
+  placement?: Placement,
```

Comment:
I think `noFlip` is missing, but I'm not sure how Flow doesn't complain. 🤔

Reply:
it does not complain because of the default value 😞